src/networks.ts

```diff
@@ -14,17 +14,7 @@ export const networkInfo = new Map<number, NetworkInfo>([
       name: 'Ethereum',
       shortName: 'eth',
       currencySymbol: 'ETH',
-      baseAPI: 'https://safe-transaction.gnosis.io',
-    },
-  ],
-  [
-    4,
-    {
-      chainID: 4,
-      name: 'Rinkeby',
-      shortName: 'rin',
-      currencySymbol: 'RIN',
-      baseAPI: 'https://safe-transaction.rinkeby.gnosis.io',
+      baseAPI: 'https://safe-transaction-mainnet.safe.global',
```

Comment:
Suggestion: fetching this from the config service.

Reply:
I'd keep it out of scope for this PR.